script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -246,17 +261,18 @@ For some reason, I specifically avoided asking 'Do you like curry?'
 [sha:790c9f2e7aaefbfc141b19a89991d199608d4da1]{
 -- Page 68, Offset 3778.
 -- 良かった……先輩も何が異常なのか察したようだ。
-Good... At least it looks like she's aware of it.
+Good... At least she seems to understand that it's slightly strange behavior.
 }
 [sha:a5e7c4afebd1449878671cf0cb6f206b3258e22d]{
 -- Page 69, Offset 3779.
 -- 「でも分かりますよ。好きなものってちょっと多いぐらい買い込んでおかないと不安になりますからね。
-"But I understand. If you don't buy enough of what you like, you get uneasy.
+// Is the idea of 不安 here that you buy a collection of bread for lunch and your date picks the one you were angling for?
+"Still, I get it. It's easy to feel nervous if you don't stock up on the things you enjoy.
```

Comment:
I understand this feeling perfectly lol, it comes from a place of insecurity, the idea that you don't deserve good things. If a lot of good things happen to you, you get the feeling that you're going to receive karmic retribution and something bad will happen. In this case, buying stuff that you like invites that sense of incoming misfortune, is what they mean.

Reply:
actually, you are right :aichaneheh: